pkgs/development/libraries/libhdhomerun/default.nix

```diff
@@ -1,12 +1,12 @@
-{ stdenv, fetchgit }:
+{ stdenv, fetchurl }:
 
-stdenv.mkDerivation {
-  name = "libhdhomerun-1efbcb";
+stdenv.mkDerivation rec {
+  pname = "libhdhomerun";
+  version = "20200521";
```

Comment:
```suggestion
  version = "2020-05-21";
```

Reply:
If you look at https://www.silicondust.com/support/downloads/ and the ubuntu and arch repos they're all referring to the release as YYYYMMDD. To comply with upstream I think we should keep it that way.